### PR TITLE
Developer navigation implementaion

### PR DESCRIPTION
**Pull request by leechristophermurray**
The developer navigation bar has been implemented. When a developer logs in, they will see options to navigate to the developer dashboard, docmentation, ABIS home page, and their user related menu.

### CODE REVIEW

**Comment by matjames007** [comment]
What is the purpose of this pull request? Change the description and the title to reflect this. The description should indicate the issue that this is resolving or the feature that this pull request will introduce. A description of the approach is also useful.

**Comment by thinkking** [comment]
I just updated the information as requested.

File: public/stylesheets/style.css [hunk 1]
```diff
@@ -1,547 +1,1334 @@
...
+    text-align: right;
+    border-top: none;
+}
+
+.disabled-app{
+    opacity: 0.4;
+}
+
+#delete-app-txt{
+    padding-left:20px;
```
Comment (leechristophermurray) [inline public/stylesheets/style.css:1333]: @CamiMonte had some style updates I added.

**Review by leechristophermurray — COMMENTED** [review]
This feature was included in a branch worked on by @nick22891, so this pull request is no longer needed.

**Pull request closed without merging**